airflow/providers/google/suite/hooks/drive.py

```diff
@@ -128,31 +128,45 @@ def get_media_request(self, file_id: str) -> HttpRequest:
         request = service.files().get_media(fileId=file_id)
         return request
 
-    def exists(self, folder_id: str, file_name: str, drive_id: Optional[str] = None):
+    def exists(
+        self, folder_id: str, file_name: str, include_trashed: bool = True, drive_id: Optional[str] = None
```

Comment:
Can we change the ordering to put `include_trashed` at the back? Right now this (unnecessarily) breaks backward compatibility if the user passes `drive_id` as a positional argument.

Reply:
Good point, I missed that one. Thanks for letting me know :)
Also, a good point about making it keyword-only.

I've implemented your suggestion :)